.github/workflows/tests.yml

```
@@ -0,0 +1,43 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [12.x, 13.x, 14.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Cache node_modules
+      uses: actions/cache@v2
+      with:
+        path: '**/node_modules'
+        key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
```

If you're going to cache, this needs a little tweaking. You need each node version to use a different cache, so the matrix node version needs including into the key. Also, ideally you pull a recent cache if the current cache isn't available. You can do that by using the restore key.
```suggestion
        key: ${{ runner.os }}-${{ matrix.node-version }}-modules-${{ hashFiles('**/yarn.lock') }}
        restore-keys: ${{ runner.os }}-${{ matrix.node-version }}-modules
```

---

Actually, it looks like you're caching node_modules. According to GitHub Actions docs:
>Note: It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci

I thought because we were using yarn that we could use the setup here to cache the packages in the yarn cache?

https://github.com/actions/cache/blob/main/examples.md#node---yarn-2

---

I didn't manage to make the cache work and since the build step is only taking ~30s I think we could merge this without the cache part. WDYT?

---

💯